### PR TITLE
Added Zone support

### PR DESCRIPTION
**Pull request by MaxGroenenboom**
Added zone support. The UpdateTo1.0.9 branch was also pulled into this branch, so please first review and merge #11 to avoid this PR merging the 1.0.9 update into the master.

### CODE REVIEW

File: src/DUWOagent/events.mod2g [hunk 1]
```diff
@@ -18,4 +18,7 @@ module events {
 	%% Percepts a list of all indicators ([<indID>, <currentValue>, <targetValue>]).
 	if percept(indicators(X)), bel(indicators(Y))
 			then delete(indicators(Y)) + insert(indicators(X)).
+	%% Percepts a list of all zones ([-todo-]).
```
Comment (Denpeer) [inline src/DUWOagent/events.mod2g]: why does it still say todo?

File: src/DUWOagent/events.mod2g [hunk 2]
```diff
@@ -23,15 +23,17 @@ module events {
 	%% Percepts a list of all buildings ([building(<id>, <name>, [<category>], <timeState>)]).
 	if percept(buildings(X)), bel(buildings(Y))
 			then delete(buildings(Y)) + insert(buildings(X)).
-
+	%% Percepts a list of all zones ([-todo-]).
```
Comment (Denpeer) [inline src/DUWOagent/events.mod2g]: why does it still say todo?

**Comment by Denpeer** [comment]
wouldn't it be a lot nicer to handle zones the same way indicators are now handled?

**Comment by MaxGroenenboom** [comment]
@Denpeer True, but wouldn't that be the case for the buildings, the settings and the functions percepts as well?

File: src/DUWOagent/events.mod2g (hunk 2 above)
Comment (MaxGroenenboom) [inline src/DUWOagent/events.mod2g]: Yeah I forgot to add that, I will do that tomorrow...

**Comment by Denpeer** [comment]
yeah, but those are still left over from wouter's implementation. I'll make a note to add refactoring these  to a future sprint.

**Comment by MaxGroenenboom** [comment]
I could fix all of them when this PR and #11 are merged?

**Comment by Denpeer** [comment]
sounds good

File: src/tests/testPercepts.test2g [hunk 3]
```diff
@@ -43,11 +43,11 @@ test events with
 		% Check if we have at least one building.
 		eventually bel(buildings([_|_])).
 
-				
+		% Check if we have at least one building.
```
Comment (klikooo) [inline src/tests/testPercepts.test2g]: This comment is exact the same as on line 43, is this correct?

**Comment by klikooo** [comment]
Should I add a predicate (or function) to create multipolygons with just x,y width and height?

**Comment by MaxGroenenboom** [comment]
Yes please, that would look a lot better than the current version

**Comment by klikooo** [comment]
Looks good now,I approve

**Comment by Denpeer** [comment]
approved!

**Pull request merged**